a10_octavia/tests/unit/controller/worker/tasks/test_a10_database_tasks.py

```diff
@@ -124,6 +124,18 @@ def test_get_vthunder_by_loadbalancer(self):
         vthunder = mock_get_vthunder.execute(LB)
         self.assertEqual(vthunder, None)
 
+    def test_get_backup_vthunder_by_loadbalancer(self):
+        self.conf.config(
+            group=a10constants.A10_GLOBAL_CONF_SECTION,
+            use_parent_partition=True)
+        mock_get_vthunder = task.GetVThunderByLoadBalancer()
+        mock_get_vthunder.vthunder_repo = mock.MagicMock()
+        mock_get_vthunder.vthunder_repo.get_vthunder_from_lb.return_value = None
+        VTHUNDER.role = "BACKUP"
```

Comment:
we usually take copy of the global objects, if we need to change those object  in specific tests. so that those changes should not cause side-effects in further test-cases.